src/PointwiseFunctions/AnalyticData/GrMhd/OrszagTangVortex.hpp

```diff
@@ -67,6 +69,17 @@ class OrszagTangVortex : public MarkAsAnalyticData {
       "The relativistic Orszag-Tang vortex"};
 
   OrszagTangVortex();
+  OrszagTangVortex(const OrszagTangVortex& /*rhs*/) = delete;
+  OrszagTangVortex& operator=(const OrszagTangVortex& /*rhs*/) = delete;
```

Comment:
I think you can `default` these, no?

Reply:
Yes, but I'll change all the RelEuler and GRMHD ones in a separate commit :)